src/pages/Protect/Protect.jsx

```diff
@@ -26,7 +26,7 @@ const Protect = (props) => {
    * @param {the value  of WETH that user want to approve} _value
    * @param {the token  address of WETH} _token
    */
-  const approve_WETH = async (_value, _token) => {
+  const approveWETH = async (_value, _token) => {
```

Comment:
Just pointing out a possible bug origin:
Wherever these functions are called, wouldn't you want to change the names there too ??

Reply:
Do correct me if I'm wrong.